src/main/kotlin/javabot/JavabotConfig.kt

```diff
@@ -57,4 +57,7 @@ interface JavabotConfig : Config, Preprocessor {
 
     @Key("javadoc.jdk.file")
     fun jdkJavadoc(): String
+
+    @Key("javabot.openweathermap.token")
```

Comment:
doesn't this support the notion of optional?

Reply:
No, but it DOES support the concept of default values, which I've changed to be an invalid key.